creusot-contracts/src/lib.rs

```diff
@@ -379,7 +379,7 @@ mod base_prelude {
     pub use crate::{
         fn_pure::FnPure,
         ghost::Ghost,
-        logic::{Int, OrdLogic, Seq, ops::IndexLogic as _},
+        logic::{Int, OrdLogic, Seq, ops::IndexLogic as _, ops::NthBitLogic as _},
```

Comment:
Do we really want to include this in the prelude? It does not seem so common to me.